app/src/main/java/com/worldsnas/starplayer/model/GetAllMusic.kt

```diff
@@ -0,0 +1,6 @@
+package com.worldsnas.starplayer.model
+
+interface GetAllMusic {
```

Comment:
esme ino ham tebghe onyeki bezarin (be comment haye ghabli tavajoh konin)
va baraye piade sazish mitnonin az `impl+(interfaceName)` estefade konin